packages/terra-paginator/src/ControlledPaginator.jsx

```diff
@@ -96,7 +96,7 @@ class Paginator extends React.Component {
       pageButtons.push((
         <a
           aria-label={`Page ${val}`}
-          aria-current={val === selectedPage && 'page'}
+          aria-current={val === selectedPage && 'true'}
```

Comment:
just curious - does `aria-current` expect the string `'true'`, or does the boolean work?

Reply:
I think we may need to keep it like this. It's an enumeration actually. https://tink.uk/using-the-aria-current-attribute/

Reply:
It works! Changed it.

Reply:
yay!